pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+- [AsyncAPI CLI](https://github.com/asyncapi/cli#installation)
+- [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation)
+
+# Create AsyncAPI documents
+An AsyncAPI document is a machine-readable definition of your Event-Driven API that can be used to generate documentation and code. You can create an AsyncAPI document with the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document requires two different pieces:
+1. Validation against the specification.
+2. Validation against the best practices or company governance rules.
+
+## Validate against specification
+Let's discuss options for validating against the specification. (This ensures that every content of the document is written in accordance with the AsyncAPI specification.) 
+
+There are several tool options you may select for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. 
+
+(It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.)
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages.
+
+When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in automation:
+
+ ```
+ asyncapi validate asyncapi.yaml
+
+ ```
+
+### Parsers (code) validation 
+AsyncAPI provides official [JavaScript](https://github.com/asyncapi/parser-js) and [Go](https://github.com/asyncapi/parser-go) parsers for validating the JSON schema in your AsyncAPI documents. 
+
+<Remember>
+If you do not run your schemas through the parsers, the JSON schemas alone cannot complete the entire validation. 
+
+Learn more about [custom JSON Schemas validation needs](https://github.com/asyncapi/spec-json-schemas#custom-validation-needs).
+</Remember>
+
+## Validation against company best practices or governance
+Now let's discuss options for validating against company best practices or governance.
+
+When AsyncAPI is used by various members of the team in an organization, API management policies can be applied to your messages before they arrive to your broker.
```

Comment:
what do you mean? is it spectral related?

Reply:
sorry @derberg, what? I didn't understand this last question, sorry 😬😬😬😬

Reply:
the thing is that this sentence sounds to me like this section is related to messages validation in runtime. But this section is supposed to be about governance. And good governance is when you know your documents are wrong before you even enter the runtime and start sending messages. And this should be about AsyncAPI documents validation, not just messages

how about
`When AsyncAPI is used by various teams, you want to make sure they follow the same rules and are consistent across the organization. It is not enough to validate AsyncAPI documents against official specification rules. An example would be that officially, `summary` property is optional, but in your organization, you want to make sure that it is always provided. You need a solution that enables you to enforce internal rules on AsyncAPI documents' providers`

makes sense?